custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -5,22 +5,20 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.1/css/bootstrap-combined.min.css"/>
```

Comment:
Did this end up not working with compress?

Reply:
It did not. failed with a `KeyError` for `'rel'`

Reply:
okay, good to know

Reply:
actually i wonder if adding `rel='stylesheet'` would fix that... will check

Reply:
Ok that fails with a more sensible "UncompressableFileError at /a/icds-cas/icds_dashboard/: '//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.1/css/bootstrap-combined.min.css' isn't accessible via COMPRESS_URL ('/static/') and can't be compressed"

Reply:
ha, at least that error makes more sense